Estaer/Converge_LevelUp
# #155820645 Create details activity layout

#### What does this PR do?
- Create details activity layout

#### Description of tasks to be completed
- Create details activity layout. The following components are displayed;
User’s profile image
User’s GitHub username
User's Organisation

#### How can this be manually tested
- Clone project
- Confirm the presence of  the following files: 
`activity_main.xml`
`activity_developer_details.xml`
`item_developer_list.xml`
`MainActivity.java`
`DeveloperAdapter.java`
`DeveloperModel.java`
- Run the emulator to view the Developers' details.

#### Screenshots
<img width="323" alt="screen shot 2018-11-21 at 18 49 41" src="https://user-images.githubusercontent.com/22410594/48852637-dcdf7000-edbe-11e8-88d8-d1a22befcf6f.png">


#### What are the relevant pivotal tracker stories
[#155820645](https://www.pivotaltracker.com/story/show/155820645)
